ProjectLighthouse.Servers.Website/Pages/AnnouncePage.cshtml

```diff
@@ -37,11 +37,15 @@
     @foreach (WebsiteAnnouncementEntity announcement in Model.Announcements)
     {
         <div class="ui blue segment" style="position: relative;">
-            <h3>@announcement.Title</h3>
-            <p style="white-space: initial;">
-                @announcement.Content
-            </p>
-
+            <div>
+                <h3>@announcement.Title</h3>
+                <div style="padding-bottom: 2em;">
+                    <span style="white-space: pre-line; ">@announcement.Content</span>
+                </div>
+                <div class="ui tiny bottom left attached label">
+                    Posted by <span style="color: black">@announcement.Publisher</span>
```

Comment:
this could have an avatar, and link back to the users profile

Reply:
Avatar messes with the styling, though I'm open to linking back to the user.